src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,222 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    public static final String LONG_LINE = "______________________________________________________________________";
+    private static final String GREETING = "Hello! I'm Duke";
+    private static final String REQUEST = "What can I do for you?";
+    private static final String EXIT = "Bye. Hope to see you again soon!";
+    private static final String ADDED_TASK = "Got it. I've added this task:";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void printGreeting() {
+        System.out.println(LONG_LINE + System.lineSeparator() + GREETING);
+        System.out.println(REQUEST + System.lineSeparator() + LONG_LINE);
+    }
+
+    public static void insertTask() throws FileNotFoundException {
+        FileAccess.readFileContents("taskdata.txt", tasks);
```

Comment:
You could consider storing the file name in a variable or a constant, this makes is easier for future updates if a developer intends to make changes to it